insights/specs/default.py

```diff
@@ -875,6 +875,13 @@ def sap_sid_num(broker):
         sap = broker[Sap]
         return [(sap.sid(i).lower(), sap.number(i)) for i in sap.local_instances]
 
+    @datasource(Sap)
+    def sap_sid_name(broker):
+        sap = broker[Sap]
```

Comment:
Please add a docstring to this function so that it will have documentation when accessed from the [Datasource Catalog](https://insights-core.readthedocs.io/en/latest/specs_catalog.html) page.

Reply:
@bfahr - Done, Please check it again. Btw, I added docstring for the other 2 too.